### ISSUE TITLE
Dice score of the TransCASCADE lower than the paper in the synapse dataset

### ISSUE THREAD

**Issue by hankyul2** [SLDGroup/CASCADE#1]
Hi, thank you for sharing your great work!

I can not reach the dice score of 82.68 as reported in table 1 of the [paper](https://openaccess.thecvf.com/content/WACV2023/papers/Rahman_Medical_Image_Segmentation_via_Cascaded_Attention_Decoding_WACV_2023_paper.pdf) with TransCASCADE on the synapse dataset. Are there any tips to reproduce your work?

Train procedures:
1. download the synapse dataset and preprocess the dataset as guided in the readme.
2. download the R50+ViT-B/16 model weight pre-trained on imagenet21k dataset.
3. run the following command:
```
CUDA_VISIBLE_DEVICES=4 python -W ignore train_synapse.py 
```

Best dice scores:
- run1: 0.8152 best dice score
- run2: 0.8047 best dice score
- avg: 0.8010 score score

Evaluation graph:
![image](https://user-images.githubusercontent.com/31476895/221328678-b28b0b10-57f7-40df-8a29-2dce69f90b3e.png)

Environment:
```
Python 3.8.12
Pytorch 1.11.0
torchvision 0.12.0
```

Thank you

Hankyul

**Comment by mostafij-rahman** [comment]
We are happy to see you are interested in CASCADE.

You can download the updated trainer.py or modify the weights for (CE and DICE) loss from (0.5 and 0.5) to (0.3 and 0.7). Also we recommend you to use the same experimental settings as ours.

We appreciate your feedback.

Best,
Mostafij

**Comment by hankyul2** [comment]
Thank you for your reply.

By following your suggestion, I could achieve the mean dice score of 82.68 as reported in the paper.

Thank you.

Hankyul
